apps/files/src/files-app-settings.js

```diff
@@ -52,6 +52,7 @@ window.addEventListener('DOMContentLoaded', function() {
 		appSettingsHeader.addEventListener('click', e => {
 			const opened = e.currentTarget.children[0].classList.contains('opened')
 			OCA.Files.Settings.settings.forEach(e => opened ? e.close() : e.open())
+			e.currentTarget.children[0].setAttribute('aria-expanded', opened ? 'false' : 'true')
```

Comment:
> Are there any objections to fix this for NC25 / merge? (As the work is already done sweat_smile )

I would suggest to be a little more careful on this line and make sure it cannot fail

```suggestion
			e.currentTarget?.children?.[0]?.setAttribute?.('aria-expanded', opened ? 'false' : 'true')
```